.pre-commit-config.yaml

```diff
@@ -15,35 +16,50 @@ repos:
       - id: name-tests-test
       - id: check-ast
       - id: debug-statements
+
```

Comment:
This doesn't look like something yamllint would require, it doesn't seem related to the change.

Reply:
Could you, please, clarify? Do you mean empty row?

Reply:
Tables have rows. This is a line. But yes, I mean that adding an empty line doesn't seem to be related to changing yamllint config. It seems to be a formatting change while the PR states that it's targeted at changing some unspecified linter configuration.